contracts/token/ERC1155/extensions/ERC1155Pausable.sol

```diff
@@ -17,7 +17,7 @@ import {Pausable} from "../../../security/Pausable.sol";
  * addition to inheriting this contract, you must define both functions, invoking the
  * {Pausable-_pause} and {Pausable-_unpause} internal functions, with appropriate
  * access control, e.g. using {AccessControl} or {Ownable}. Not doing so will
- * make the contract unpausable.
+ * make the contract not to have pausable capabilities at all.
```

Comment:
Minor suggestion:

```suggestion
 * make the pause mechanism of the contract unreachable, and thus unusable.
```

Reply:
I also realized this changed had to be made on `ERC20Pausable` and `ERC721Pausable` so I implemented the same suggestion there.